doc/pachub/pachub_getting_started.md

```diff
@@ -0,0 +1,211 @@
+# Getting Started with Pachyderm Hub
```

Comment:
Not sure if or where to add it, but might be good to say "Pachyderm Hub is in beta and we'd love your feedback and help you troubleshoot any issues or confusion. Please join our public slack channel."

I know its already 10 other places, but we especially want to make sure Hub users get funneled there so we can get feedback.